### ISSUE TITLE
How do I get this to work with Plex if I don't use XMBC? (importer agent already installed)

### ISSUE THREAD

**Issue by berrysoda** [DoctorD1501/JAVMovieScraper#314]
I have the scraper program working, I have XBMCnfoMoviesImporter installed. I assume I use the match feature with XBMCnfoMoviesImporter, but where do I put the nfo file and images so that it works?

https://forums.plex.tv/t/metadata-agents-for-exported-xbmc-library/15430 implies I already have an XMBC library set up, but I don't imagine I NEED to. Do I need to create some specific folder to put the metadata in? "This will create .nfo & .jpg files for your TV Shows and Movies, **in the appropriate directories**."

Can I just manually create said directories?

TLDR: Where does XBMCnfoMoviesImporter pull the metadata from on my computer?

**Comment by Still34** [comment]
Why is this labeled as `bug` 🤔

Anyways, the nfo and images go directly into the folder of matching video. For example, you have a video with an ID of `AAAA-000`, you'd put the video in a folder with the same ID, e.g. `/videos/AAAA-000`, then have the scraper scrape the folder, write to file, and refresh metadata in Plex and it should pick up the NFO and poster files.

**Comment by berrysoda** [comment]
Alright, thanks. I think I'm figuring things out. 

Is there any way to get it to either not pull the title, or to pull the title (and filename) as <ACTORS> - <ID> ? That's the last thing I'm trying to figure out. Disabling everything for title pulling doesn't seem to actually disable it

**Comment by Still34** [comment]
Not really no. It will try to fill every field possible.
